PluralKit.Bot/Commands/MemberAvatar.cs

```diff
@@ -131,6 +177,10 @@ public async Task Avatar(Context ctx, PKMember target)
                 $" This avatar will now be used when proxying in this server (**{ctx.Guild.Name}**).",
             AvatarLocation.Member when targetGuildData?.AvatarUrl != null =>
                 $"\n{Emojis.Note} Note that this member *also* has a server-specific avatar set in this server (**{ctx.Guild.Name}**), and thus changing the global avatar will have no effect here.",
+            AvatarLocation.MemberWebhook when targetGuildData?.AvatarUrl != null =>
+                $" This avatar will now be used for this member's proxied messages, instead of their main avatar.\n{Emojis.Note} Note that this member *also* has a server-specific avatar set in this server (**{ctx.Guild.Name}**), and thus changing the global avatar will have no effect here.",
```

Comment:
generally for notes like this we send a second message

Reply:
That exact note is sent in the same message on a new line directly above this change for normal avatars when a server avatar is set, so IMO this is fine?